lib/chef/win32/security.rb

```diff
@@ -341,6 +341,21 @@ def self.get_token_information_primary_group(token)
         SID.new(group_result[:PrimaryGroup], group_result_storage)
       end
 
+      def self.get_token_information_elevation_type(token)
+        token_result_size = FFI::MemoryPointer.new(:ulong)
+        if GetTokenInformation(token.handle.handle, :TokenElevationType, nil, 0, token_result_size)
+          raise "Expected ERROR_INSUFFICIENT_BUFFER from GetTokenInformation, and got no error!"
```

Comment:
Why is this check needed?

Reply:
@jaym , `GetTokenInformation` method does return `ERROR_INSUFFICIENT_BUFFER` error if the input provided to the method isn't correct. It's not mandatory to keep this check, but I have made it consistent with other existing methods e.g. https://github.com/chef/chef/blob/master/lib/chef/win32/security.rb#L317 and https://github.com/chef/chef/blob/master/lib/chef/win32/security.rb#L332.
Please let me know your thought on that.